app/enums/IncorporationStatus.scala

```diff
@@ -24,7 +25,12 @@ object IncorporationStatus extends Enumeration {
 
   implicit val format = Format(Reads.enumNameReads(IncorporationStatus), Writes.enumNameWrites)
```

Comment:
should the implicit format not take in the writes from the def i.e. 
implicit val format = Format(Reads.enumNameReads(IncorporationStatus), writes(APIValidation))